testing/qt5-qtserialport/APKBUILD

```diff
@@ -0,0 +1,43 @@
+# Contributor: Denis Ryabyy <vv1r0x@gmail.com> 
+# Maintainer: Denis Ryabyy <vv1r0x@gmail.com>
+pkgname=qt5-qtserialport
+pkgver=5.6.0
+pkgrel=1
+pkgdesc="Interface to serial ports for Qt application"
+url="http://qt-project.org/wiki/QtSerialPort"
+arch="all"
+license="LGPL"
+depends="qt5-qtbase"
+depends_dev="$depends"
+makedepends="$depends_dev qt5-qtbase-dev"
+install=""
+subpackages="$pkgname-dev"
+
+_ver=${pkgver/_/-}
+_ver=${_ver/beta0/beta}
+_ver=${_ver/rc0/rc}
+_V=${_ver/rc/RC}
```

Comment:
```
+_ver=${pkgver/_/-}
+_ver=${_ver/beta0/beta}
+_ver=${_ver/rc0/rc}
+_V=${_ver/rc/RC}

```
I got it from qt5-qtbase
https://github.com/alpinelinux/aports/blob/1b77b99057f204718bab6468f87b16ac07190aae/main/qt5-qtbase/APKBUILD